mod/admin.php

```diff
@@ -1441,11 +1441,11 @@ function admin_page_dbsync(App $a)
  */
 function admin_page_users_post(App $a)
 {
-	$pending     = (x($_POST, 'pending')           ? $_POST['pending']           : array());
-	$users       = (x($_POST, 'user')              ? $_POST['user']		      : array());
-	$nu_name     = (x($_POST, 'new_user_name')     ? $_POST['new_user_name']     : '');
-	$nu_nickname = (x($_POST, 'new_user_nickname') ? $_POST['new_user_nickname'] : '');
-	$nu_email    = (x($_POST, 'new_user_email')    ? $_POST['new_user_email']    : '');
+	$pending     = defaults($_POST, 'pending'          , array());
```

Comment:
Just a style question: Wouldn't it be preferable to set the comma directly after the parameter and not with much space before?

Reply:
To align with the previous `defaults()` calls default values.